### ISSUE TITLE
trained model and configs

### ISSUE THREAD

**Issue by felix-ky** [garvita-tiwari/PoseNDF#19]
Thank you for updating the code, 

could you please update "amass_softplus.yaml" and the optimal model?

best wishes

**Comment by garvita-tiwari** [comment]
We are running some tests and will update the model and config asap.
Sorry for the delay

**Comment by felix-ky** [comment]
Thank you so much.

Do denoise results of humor in Table 1 computed by adding noise to rotation and then converted to 3Djoints as input to humor's TestOpt? as I found humor only performs denoising of 3Djoints. 

best wishes

**Comment by garvita-tiwari** [comment]
> Thank you so much.
> 
> Do denoise results of humor in Table 1 computed by adding noise to rotation and then converted to 3Djoints as input to humor's TestOpt? as I found humor only performs denoising of 3Djoints.
> 
> best wishes

Hi, we use the loss function described in eq 7 and 8 of the paper(section 4.2), we use both joint and SMPL vertices in our denoising.

**Comment by felix-ky** [comment]
Thank you so much 

to confirm, denoising using humor in table 1 just change posndf prior to humor prior in eq 7 ?
could you update the code using humor to denoise?

best wishes
